apps/api/src/server/applications/application/__tests__/update-document-status.test.js

```diff
@@ -59,12 +59,15 @@ test('updates document status', async (t) => {
 		guid: 'D1234',
 		status: 'awaiting_virus_check'
 	});
+	// below test not working
```

Comment:
did this break?

Reply:
yes. There were many missing dependencies and even missing functions but I think I fixed them. This test is the only thing that I'm not able to fix